docker-compose.yaml

```diff
@@ -68,33 +54,24 @@ services:
         sawtooth-validator --endpoint tcp://validator:8800 -v \
             --bind component:tcp://eth0:4004 \
             --bind network:tcp://eth0:8800 \
-    \""
-    stop_signal: SIGKILL
+      \""
 
   rest-api:
-    image: hyperledger/sawtooth-rest-api:latest
-    expose:
-      - 4004
-      - 8080
+    image: hyperledger/sawtooth-rest-api:1.0.1
     ports:
       - 8080:8080
     depends_on:
      - validator
     command: sawtooth-rest-api -v --connect tcp://validator:4004 --bind rest-api:8080
-    stop_signal: SIGKILL
 
-  seth:
-    container_name: seth
+  seth-rpc:
     image: sawtooth-seth:latest
-    volumes:
-      - ./:/project/sawtooth-seth
-    expose:
-      - 3030
-      - 8080
     ports:
       - 3030:3030
-    working_dir: "/project/sawtooth-seth"
-    command: "bash -c \"
+    depends_on:
```

Comment:
This depends_on is not needed, and really has no impact since depends_on has no ability to determine when services inside a container are actually ready.

Reply:
Fair enough